.github/workflows/release.yml

```diff
@@ -77,3 +77,4 @@ jobs:
           export OWNER=$( echo ${{ github.repository }} | cut -d "/" -f 1 )
           gem build station.gemspec
           gem push --KEY github --host https://rubygems.pkg.github.com/${OWNER} *.gem
+          gem push *.gem
```

Comment:
@bencgreenberg do we have the credentials in place for this?

Reply:
Based on line 73 of the file before I added this it appears that we do.